scripts/us_bls/jolts/BLSJolts_StatisticalVariables.mcf

```diff
@@ -0,0 +1,1416 @@
+
+Node: dcid:jobChangeEvent
+name: "jobChangeEvent"
+typeOf: schema:Property
+domain: schema:Person
+range: dcs:JobChangeEventEnum
+
+Node: dcid:JobChangeEventEnum
+name: "JobChangeEventEnum"
+description: "Describes different job change events that may happen to workers" 
+typeOf: dcs:Class
+subClassOf: Enumeration
+
+Node: dcid:Hired
+name: "Hired"
+description: "Describes a worker that was hired to a company"
+typeOf: dcid:JobChangeEventEnum
+
+Node: dcid:Separated
+name: "Separated"
+description: "Describes a worker that was separated from a company"
+typeOf: dcid:JobChangeEventEnum
+
+Node: dcid:VoluntarySeparation
+name: "VoluntarySeparation"
+description: "Describes a worker that voluntarily separated from a company"
+typeOf: dcid:JobChangeEventEnum
+specializationOf: dcs:Separated
```

Comment:
Ditto, but also for specializationOf, we treat as "sensitive"--they will go in [enum_specializations.mcf](https://github.com/datacommonsorg/data/blob/master/schema/enum_specializations.mcf)

Reply:
Should I remove them from here?